css/templates/travel/components/_city-selector.css

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright 2017 The AMP Start Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ /**
+ * city-selector
+ *
+ * Style an <amp-selector> element as a list of labeled checkboxes.
+ */
+.travel-city-selector {
+  margin: -6px;
+}
+
+.travel-city-selector [option] {
+  display: flex;
+  align-items: center;
+  position: relative;
+  padding: 6px;
+}
+
+.travel-city-selector [option] > span {
+  display: flex;
+  align-items: center;
+}
+
+/* Override core amp [option][selected] styles. */
+.travel-city-selector [option][selected] {
+  cursor: pointer;
+  outline: none;
+}
+
+.travel-city-selector [option] amp-img {
+  width: 38px;
+  height: 38px;
+  margin-right: 10px;
+  background: var(--silver);
+  position: relative;
+
+  transition: transform 300ms ease-in-out;
+}
+
+.travel-city-selector [option] amp-img img {
+  opacity: 0.5;
+  transition: opacity 150ms steps(1, start) 150ms, transform 150ms steps(1, start) 150ms;
+}
+
+.travel-city-selector [option]::before,
+.travel-city-selector [option]::after {
+  width: 18px;
+  height: 18px;
+  position: absolute;
+  /* Right offset is equal to the focus shadow width. */
+  right: 3px;
+  top: 0;
+  bottom: 0;
+  margin: auto;
+}
+
+
+.travel-city-selector [option]::before {
+  content: "";
```

Comment:
''

Reply:
Sorry, I'm not sure what you mean by this @camelburrito

Reply:
Never mind, I understand